### PR TITLE
Implement MongoDB

### LINKED ISSUE #16: MongoDB
#### Add connection to MongoDB

Install MongoDB and implement connection to server.

#### Tasks

- [x] Install MongoDB
- [x] Include MongoDB key in .env & gitignore
- [x] Set up connection in database.ts

### PR DESCRIPTION
**Pull request by J-Bannach**
fixes #16 

- Install MongoDB
- Connect to MongoDB
- Test read/write in MongoDB

### CODE REVIEW

File: samples.http [hunk 1]
```diff
@@ -5,7 +5,17 @@
 GET {{baseURL}}/api/search?recipeName=Arrabiata
 
 
+### Add new recipe
+POST {{baseURL}}/api/credentials HTTP/1.1
```
Comment (lmachens) [inline samples.http:9]: `credentials`? Your project is about recipes.

**Review by lmachens — CHANGES_REQUESTED** [review]

**Review by l-goede — APPROVED** [review]
lgtm!:)

**Review by daebert — APPROVED** [review]
Lgtm!

**Review by roabram — APPROVED** [review]
lgtm 💯

**Review by Niklas-Schaefers — APPROVED** [review]
lgtm 🚀

**Pull request merged**